lib/views/quiz/quiz_view.dart

```diff
@@ -72,14 +72,16 @@ class QuizViewState extends State<QuizView> {
       appBar: questionIndex < widget.quiz.questions.length
           ? AppBar(
               brightness: Brightness.dark,
-              title: Heading(widget.quiz.title),
+              title: SubHeading(widget.quiz.title),
```

Comment:
Is this going to give us adequate character count for quiz titles or do we need to do something more to fix the issue?

Reply:
I will take a look

Reply:
Im not entirely sure how to get an accurate answer "General knowledge" overflows on both note 8 and nexus S when set to Heading however when set to SubHeading neither overflow do you have any suggestions?

Reply:
I think we should make another widget called AppBarHeading, for example, that uses a small font for small devices and a larger font for large devices. We can figure out what the max character count on small devices is and limit the length to that